api/src/controllers/user.js

```diff
@@ -0,0 +1,41 @@
+import { AUDIT_ACTIONS } from '../constants'
+import InvitationRequest from '../db/models/InvitationRequest'
+import log from 'log'
+import { createAudit } from '../utils/audit'
+
+import {
+    getRequestStatuses,
+  } from '../utils/github'
+
+  export const getUserPendingInvitations = async (req, res) => {
```

Comment:
getPendingInvitations

we know that this is for users

Reply:
renamed